### DISCUSSION TITLE
Covers, cover cache and CovercachePrepopulate with version 16.0.0

### DISCUSSION THREAD

**Discussion by GK-774** [jcorporation/myMPD#1313] [Q&A] [answered]
A question about the handling of covers since 16.0.0:

For most of my albums I have a cover.jpg and cover-sm.jpg in their directories (as recommended). This works nicely with version 15.0.2, browsing the albums shows the cover-sm.jpg and album details or playing an album displays the full sized cover.
This behaviour appears to have changed in 16.0.0, album details or playing an album now displays a low resolution upscaled version of cover-sm.jpg.

Is this intentionally so or is it a result of using the CovercachePrepopulate script (which I did)?

**Comment by GK-774** [comment]
Well, it appears to be the CovercachePrepopulate script. When I clear the cover cache and do not prepopulate it with the script everything works as expected (like described for 15.0.2 above).
Basically caching all the covers (small and full size) in /var/cache, which could be on a SSD while the audio files are on a conventional disk or on a NAS. would make sense and afford a speed up.

**Reply by jcorporation** [reply, marked as answer]
The covercache does not distinct between the small or normal images.

The script actually fetch the small images. This could be rewritten to fetch the normal ones.

Edit: Fixed with https://github.com/jcorporation/mympd-scripts/commit/5318b479e15665e43e1af2222bba828bd311608c
